### PR TITLE
Cloud Changelog July 14th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 1]
```diff
@@ -2,6 +2,30 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-07-14
+
+### Highlights
+
+#### Granular Data Retention Policies
```
Comment (amyblais) [inline source/cloud/cloud-administration/cloud-changelog.md:9]: Do we want to mention this known issue Known issue: https://mattermost.atlassian.net/browse/MM-36574?

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+ - A ``data_retention`` type job can now be run even if the global policy is disabled. The granular (i.e. team and channel-specific) policies will be executed when the data_retention job is run.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a **Do Not Disturb** status submenu with the options to disable notifications for 30min, 1h, 2h, Tomorrow, or Custom.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:15]:
```suggestion
 - Added a **Do Not Disturb** availability submenu with the options to disable notifications for 30min, 1h, 2h, Tomorrow, or Custom.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 3]
```diff
@@ -2,6 +2,30 @@
...
+ - Markdown formatting is now stripped from push notifications.
+ - Improved performance of components that show reactions on posts.
+ - Improved performance of certain components when viewing non-Direct Message channels.
+ - Added minor improvements to performance of messages posted in the right-hand side.
+ - Added icons to apps in the Marketplace.
+ - Apps can now add arbitrary markdown in between fields on forms.
+ - Added support for markdown in apps forms and interactive dialogs field descriptions and errors, and slash commands.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:22]:
```suggestion
 - Added support for markdown in apps forms, interactive dialogs field descriptions, errors, and slash commands.
```

**Review by cwarnermm — APPROVED** [review]
Non-blocking suggestions provided inline

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 4]
```diff
@@ -2,6 +2,30 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-07-14
+
+### Highlights
+
+#### Granular Data Retention Policies
+ - A ``data_retention`` type job can now be run even if the global policy is disabled. The granular (i.e. team and channel-specific) policies will be executed when the data_retention job is run.
```
Comment (wiersgallak) [inline source/cloud/cloud-administration/cloud-changelog.md:10]:
```suggestion
 - A ``data_retention`` type job can now be run even if the global policy is disabled. The granular (i.e. team and channel-specific) policies will be executed when the data retention job is run. Please note there is a known issue where deleted posts get displayed in channels without new activity after the retention job is run.  This issue is tracked with `this ticket <https://mattermost.atlassian.net/browse/MM-36574>`_. 
```

File: source/cloud/cloud-administration/cloud-changelog.md (hunk 1 above)
Comment (wiersgallak) [reply source/cloud/cloud-administration/cloud-changelog.md:9]: Yeah I think that makes sense.  Added inline suggestion below.

**Review by wiersgallak — APPROVED** [review]
Thanks for fixing the syntax, Amy! Looks good to me.

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 5]
```diff
@@ -2,6 +2,30 @@
...
+ - Markdown formatting is now stripped from push notifications.
+ - Improved performance of components that show reactions on posts.
+ - Improved performance of certain components when viewing non-Direct Message channels.
+ - Added minor improvements to performance of messages posted in the right-hand side.
+ - Added icons to apps in the Marketplace.
+ - Apps can now add arbitrary markdown in between fields on forms.
+ - Added support for markdown in apps forms, interactive dialogs field descriptions, errors, and slash commands.
+
+### Known Issues
+ - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channels**.
```
Comment (esethna) [inline source/cloud/cloud-administration/cloud-changelog.md:25]: Propose adding a link to CRT docs known issues section in every cloud and on-prem changelog until we release the feature in GA

**Review by esethna — APPROVED** [review]

File: source/cloud/cloud-administration/cloud-changelog.md (hunk 5 above)
Comment (esethna) [reply source/cloud/cloud-administration/cloud-changelog.md:25]: @amyblais ^

**Review by BenCookie95 — APPROVED** [review]

**Pull request merged**